subset/network/test_network

```diff
@@ -2,14 +2,15 @@
 
 REPORT=/tmp/report.txt
 
+STARTUP=/scans/startup.pcap
 MONITOR=/scans/monitor.pcap
 MODULE_CONFIG=/config/device/module_config.json
 EXCLUDES=infastructure_excludes.json
 
-python network_tests.py connection.dhcp_long $MONITOR $TARGET_IP
-python network_tests.py connection.min_send $MONITOR $TARGET_IP
-python network_tests.py communication.type.broadcast $MONITOR $TARGET_IP
-python network_tests.py protocol.app_min_send $MONITOR $TARGET_IP $MODULE_CONFIG $EXCLUDES
-python network_tests.py network.ntp.support $MONITOR $TARGET_IP
+#python network_tests.py communication.type.broadcast $MONITOR $TARGET_IP
+#python network_tests.py connection.dhcp_long $MONITOR $TARGET_IP
+#python network_tests.py connection.min_send $MONITOR $TARGET_IP
+python network_tests.py connection.network.ntpv4 $STARTUP $TARGET_IP
+#python network_tests.py protocol.app_min_send $MONITOR $TARGET_IP $MODULE_CONFIG $EXCLUDES
```

Comment:
@frgitdaq it's best to move the ntp code in a separate ntp module (test_ntp) to avoid collisions with other tests.
Leave the previous test code as it is as it will be addressed in a separate issue and PR.

Reply:
@pisuke The code has now been moved into a separate module.